tests/integration/test_dt_webhook.py

```diff
@@ -0,0 +1,78 @@
+"""Integration tests for dt_webhook.py."""
+import asyncio
+import json
+from http import HTTPStatus
+
+import aiohttp
+
+# pylint: disable-next=import-error
+import pytest
+
+from extensions.eda.plugins.event_source.dt_webhook import main as dt_webhook
+
+args = {
+    "host": "127.0.0.1",
+    "port": 1234,
+    "token": "thisisnotanactualtoken",
+}
+url = f'http://{args["host"]}:{args["port"]}/event'
+payload = json.dumps({"eventId": "1A2B3C"})
+headers = {"Authorization": "Bearer " + args["token"]}
+
+async def run_webhook() -> None:  # noqa: FA102, D103
+    """Start webhook."""
+    await dt_webhook(asyncio.Queue(), args)
+
+@pytest.mark.asyncio
+async def test_with_incorrect_path():
+    async def do_request():
+        async with aiohttp.ClientSession(headers=headers) as session:
+            async with session.post(f'http://{args["host"]}:{args["port"]}/something', data=payload) as resp:
+                plugin_task.cancel()
+                assert resp.status == HTTPStatus.NOT_FOUND
```

Comment:
I don't know how tests with async io work, but I would expect the `assert` statement to come somewhere after the `await asyncio.gather` - though that would require `resp` to be passed to the outside.
Long story short, I think this is fine :D